docs/modules/globalfee.md

```diff
@@ -1,26 +1,28 @@
 # Gaia Fees and Fees Checks
 
 ## Fee Parameters
-The CosmosHub allows managing fees using 3 parameters:
+The CosmosHub allows managing fees requirement using 4 parameters. At the network level, there are three parameters from globalfee modules (`MinimumGasPricesParam`, `BypassMinFeeMsgTypes`, and `MaxTotalBypassMinFeeMsgGasUsage`) that can be set by gov proposal. Additionally, there's a fourth parameter that enables individual nodes to impose supplementary fee amount requirement.
 
-1. setting global fees (`MinimumGasPricesParam`)
-Global fees are defined at the network level by setting `MinimumGasPricesParam`, via [Gov Proposals](https://hub.cosmos.network/main/governance/proposals/)
-
-2. `minimum-gas-prices`
-This parameter is part of the node configuration, it can be set in the `config/app.toml` configuration file.
-
-3. `bypass-min-fee-msg-types`
-This parameter is part of the node configuration, it can be set in the `config/app.toml` configuration file.
-This represents a list of message types that will be excluded from paying any fees for inclusion in a block.
-
-Both global fees (`MinimumGasPricesParam`) and `minimum-gas-prices` represent a list of coins, each denoted by an amount and domination as defined by [sdk.DecCoins](https://github.com/cosmos/cosmos-sdk/blob/82ce891aa67f635f3b324b7a52386d5405c5abd0/types/dec_coin.go#L158) 
+1. Global fees (`MinimumGasPricesParam`).\
```

Comment:
1. Global fees (`MinimumGasPricesParam`). ~~\\~~
**The** Global fee `MinimumGasPricesParam` is established at the network level **,** through **the** global fee params set ~~ting~~ via **a** Governance Proposal~~s, and it~~ . **This** sets ~~the~~ a fee requirement~~s~~ that the entire network must adhere to.
2. `minimum-gas-prices` in `app.toml`\
   By adjusting the minimum-gas-prices parameter in app.toml, nodes can enforce a fee that is higher than the globally defined MinimumGasPricesParam. However, it's important to note that this configuration solely determines whether transactions are eligible to enter this specific node's mempool.